base_local_planner/src/latched_stop_rotate_controller.cpp

```diff
@@ -19,12 +19,13 @@
 
 namespace base_local_planner {
 
-LatchedStopRotateController::LatchedStopRotateController(const std::string& name) {
-  ros::NodeHandle private_nh("~/" + name);
-  private_nh.param("latch_xy_goal_tolerance", latch_xy_goal_tolerance_, false);
+LatchedStopRotateController::LatchedStopRotateController() {}
 
-  rotating_to_goal_ = false;
-}
+void LatchedStopRotateController::initialize(const std::string& name) {
+    ros::NodeHandle private_nh("~/" + name);
+    private_nh.param("latch_xy_goal_tolerance", latch_xy_goal_tolerance_, false);
```

Comment:
Please change this to search for the parameter as well, for people who relied on the old configuration.

Reply:
May I know which parameter are you talking about?

Reply:
You mean checking if "latch_xy_goal_tolerance" is set? Or search for the same in global namespace as well as in planner namespace?

Reply:
No, just use the standard searchParam functionality to check the `~/name` namespace as well as `~/`

Reply:
Got it. Will do.

Reply:
Did the change you had asked for.